### PR TITLE
feat: Integration testing - round 1

### PR DESCRIPTION
**Pull request by muselesscreator**
Resolves a number of console warnings about paragon component usage, and introduces a set of integration tests that go through initialization, selection, and submission navigation.  The tests verify against redux shape that the submissions are loaded into the list appropriately, selected for review appropriately, and that as you navigate through selected submissions, the "current" has the full status and response, and prev/next containers just hold the response.

### CODE REVIEW

File: src/containers/ListView/__snapshots__/ListViewBreadcrumb.test.jsx.snap [hunk 1]
```diff
@@ -18,7 +18,11 @@ exports[`ListViewBreadcrumb component component snapshot: empty (no list data) 1
     <Hyperlink
       destination="oraUrl(test-course-id, fake-location-id)"
       target="_blank"
-    />
+    >
+      <Icon
+        icon={[Function]}
```
Comment (leangseu-edx) [inline src/containers/ListView/__snapshots__/ListViewBreadcrumb.test.jsx.snap]: Are we not mocking `Icon`?

File: src/data/reducers/grading.js [hunk 2]
```diff
@@ -100,7 +100,7 @@ const app = createReducer(initialState, {
   [actions.grading.preloadPrev]: (state, { payload }) => ({ ...state, prev: payload }),
   [actions.grading.loadNext]: (state, { payload }) => ({
     ...state,
-    prev: state.current,
+    prev: { response: state.current.response },
```
Comment (leangseu-edx) [inline src/data/reducers/grading.js:103]: 👍  nice catch.

**Review by leangseu-edx — APPROVED** [review]
1 question. The rest LGTM 👍

File: src/containers/ListView/__snapshots__/ListViewBreadcrumb.test.jsx.snap (hunk 1 above)
Comment (muselesscreator) [reply src/containers/ListView/__snapshots__/ListViewBreadcrumb.test.jsx.snap]: touche

**Pull request merged**